.github/actions/setup-rocm/action.yml

```diff
@@ -5,20 +5,6 @@ description: Set up ROCm host for CI
 runs:
   using: composite
   steps:
-    - name: Set DOCKER_HOST
-      shell: bash
-      run: echo "DOCKER_HOST=unix:///run/user/$(id -u)/docker.sock" >> "${GITHUB_ENV}"
-
-    - name: Remove leftover Docker config file
-      shell: bash
-      continue-on-error: true
-      run: |
-        set -ex
-
-        cat ~/.docker/config.json || true
-        # https://stackoverflow.com/questions/64455468/error-when-logging-into-ecr-with-docker-login-error-saving-credentials-not
-        rm -f ~/.docker/config.json
-
```

Comment:
@saienduri @amdfaa We need to make sure the changes here work not just for the Mi300 nodes but also our existing CI nodes. Please discuss offline how to resolve that. Afaiu, setting DOCKER_HOST is necessary to have our existing nodes working?

Reply:
I should be able to conditionalize on the type of node that is being used. The step on line 8 appears to be necessary but I don't think 12 is necessary.

Reply:
You can add environment variables as part of the runner startup if needed for some machines:
<img width="543" alt="{22C99364-AC70-49AD-BA8C-1DDF3B8D5CB5}" src="https://github.com/user-attachments/assets/e26f8c48-87f1-4dbc-9647-d187b3b24fcb">
We shouldn't be doing all of this in the workflow files @amdfaa

Reply:
@amdfaa This seems like a reasonable suggestion, can you please update your runner setup scripts to use the `actions-runners/.env` file to set any required env variables for the runner, and update all the non-MI300 runners to have that file?
Let us know if you have any concerns.

Reply:
A big concern is to start and stop the runners. The file `actions-runners/.env` has to be set prior to starting the service.